client/app/telecom/telephony/line/calls/simultaneousLines/translations/Messages_fr_FR.xml

```diff
@@ -60,4 +60,5 @@
    <translation id="telephony_line_actions_line_calls_trunk_simultaneous_channels_price_per_month" qtlid="479837">/ par mois</translation>
    <translation id="telephony_line_actions_line_calls_trunk_simultaneous_channels_bandwidth_rate" qtlid="479850">{{bandwidth}} Kbps</translation>
 
+   <translation id="telephony_line_actions_line_calls_simultaneous_tooltip">Pour le nombre d'appels simultanés maximum par téléphone, veuillez vous référer à la documentation de votre téléphone.</translation>
```

Comment:
```suggestion
   <translation id="telephony_line_actions_line_calls_simultaneous_tooltip">Pour le nombre d'appels simultanés maximum par téléphone, veuillez-vous référer à la documentation de votre téléphone.</translation>
```
Also something like `Le nombre d'appels simultanés est dépendant de votre matériel, veuillez-vous référer à sa documentation plus d'informations.` would be better, don't you think ?

Reply:
yes, it's better.